flows/test/quickstart and docker tutorial/3-prefect_basic_agent.py

```diff
@@ -0,0 +1,32 @@
+from prefect import task, Flow
+from prefect.run_configs import LocalRun
+
+# Start your local agent with:
+# prefect agent local start -l youragentname -l local
```

Comment:
if you add `--no-hostname-label` here it will not include your host's name as a tag in the agent

Reply:
Thanks Chia! I think local agents automatically add the hostname so this will overwrite that!